istioctl/cmd/wait.go

```diff
@@ -95,13 +95,13 @@ func waitCmd() *cobra.Command {
 			for {
 				// run the check here as soon as we start
 				// because tickers won't run immediately
-				present, notpresent, err := poll(cmd, generations, targetResource, opts)
+				present, notpresent, sdcnum, err := poll(cmd, generations, targetResource, opts)
 				printVerbosef(cmd, "Received poll result: %d/%d", present, present+notpresent)
 				if err != nil {
 					return err
 				} else if float32(present)/float32(present+notpresent) >= threshold {
-					_, _ = fmt.Fprintf(cmd.OutOrStdout(), "Resource %s present on %d out of %d sidecars\n",
-						targetResource, present, present+notpresent)
+					_, _ = fmt.Fprintf(cmd.OutOrStdout(), "Resource %s present on %d out of %d configurations for totally %d sidecars\n",
```

Comment:
`Resource %s present for %d out of %d resources for %d sidecars\n"` may be a bit more clear

Reply:
@howardjohn Thanks, it has been updated in the new PR.

Reply:
Is modifying the message part of the fix?

Reply:
@therealmitchconnors no, it's not